modular_skyrat/modules/opposing_force/code/opposing_force_equipment.dm

```diff
@@ -0,0 +1,572 @@
+/**
```

Comment:
The formatting in this file isn't consistent, I know it's just a matter of style but it makes it feel all the more clunkier.